west.yml

```diff
@@ -40,10 +40,10 @@ manifest:
     - name: fw-nrfconnect-zephyr
       path: zephyr
       west-commands: scripts/west-commands.yml
-      revision: 3c4f27282002e634d60216f23fb341317a4b199f
+      revision: pull/147/head
```

Comment:
I've just +1ed this one

Reply:
Thanks, FYI, we're waiting now for the nrf_desktop team to update their project and fix the CI, there were some API changes upstream (settings mostly).